src/assets/scripts/layout/app.js

```diff
@@ -57,6 +60,24 @@ var page = {
             page.windowHeight = height;
         }
     },
+    cookieController: () => {

```

Comment:
Let's cookieControler into a dedicated module structured like the rest. then the init function should do the accepts cookie. also rather than pudding the whole callback int the click call pull that into its own function on the cookie module.  Other than that it looks good.